docker/rocm/Dockerfile

```diff
@@ -70,10 +70,10 @@ RUN apt-get -y install libnuma1
 WORKDIR /opt/frigate/
 COPY --from=rootfs / /
 
-COPY docker/rocm/requirements-wheels-rocm.txt /requirements.txt
-RUN python3 -m pip install --upgrade pip \
-    && pip3 uninstall -y onnxruntime-openvino \
-    && pip3 install -r /requirements.txt
+#COPY docker/rocm/requirements-wheels-rocm.txt /requirements.txt
```

Comment:
Can these commented lines just be removed?

Reply:
I can add a note to the comment, but this is only temporary while I try to see if I can build one for py311

Reply:
Cool.